pkgs/development/python-modules/lap/default.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildPythonPackage,
+  cython,
+  fetchPypi,
+  numpy,
+  pytestCheckHook,
+  python-utils,
+  pythonOlder,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "lap";
+  version = "0.5.12";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    pname = "lap";
+    inherit version;
+    hash = "sha256-VwtBTqeubAS9SdDsjNrB3FY0c3dVeE1E43+fZourRP0=";
+  };
+
+  build-system = [ setuptools ];
+
+  nativeBuildInputs = [ cython ];
+
+  dependencies = [
+    numpy
+    python-utils
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "lap" ];
+  preCheck = ''
+    cd "$out"
+  '';
+
+  meta = with lib; {
+    description = "Linear Assignment Problem solver (LAPJV/LAPMOD). ";
+    homepage = "https://github.com/gatagat/lap";
+    changelog = "https://github.com/gatagat/lap/releases/tag/v${version}";
+    license = licenses.bsd2;
+    maintainers = [ ];
+  };
+}
```

Someone has to maintain this package, add me, and yourself if you have the means for that. Also, note the `with lib;` reduction.

```suggestion
  meta = {
    description = "Linear Assignment Problem solver (LAPJV/LAPMOD). ";
    homepage = "https://github.com/gatagat/lap";
    changelog = "https://github.com/gatagat/lap/releases/tag/v${version}";
    license = lib.licenses.bsd2;
    maintainers = with lib.maintainers [ doronbehar tebriel ];
  };
}
```

I guessed your `lib.maintainers` attribute of course.